node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -1026,84 +1103,48 @@ mod tests {
 
```

Comment:
If you want to keep the line
`type Subject = BlockchainInterfaceWeb3;`,
I'd move it probably to the absolute top. It's not an actual expression or statement, it's just a type alias. Strange use anyway.